build.py

```diff
@@ -67,7 +67,7 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.30.0': (
+    '2.30.0dev': (
```

Comment:
Is dev required? I believe dev is only required on `main` branch.

Reply:
I will remove this commit once CI passes and before merging the PR. https://github.com/triton-inference-server/server/pull/5340/commits/641096b761f7cbe4689aa08adb5dd4f69ca7657b
The 23.01-devel gitlab CI expects dev.